packages/fast-components-react-base/src/listbox-item/listbox-item.tsx

```diff
@@ -24,25 +25,67 @@ class ListboxItem extends Foundation<
         disabled: false,
     };
 
+    /**
+     * Gets the id to use for the item
+     * Authors who's items all have unique values can choose to
+     * not provide an itemId and the values of the items will be used as id's,
+     * otherwise use itemId.  All listbox items of a particular listbox should use the same
+     * approach with respect to using values or itemId or results could be unexpected.
+     * The "id" prop is being depracated and should not be used.
+     */
+    public static getItemId = (props: ListboxItemProps): string => {
+        return !ListboxItem.isNullOrUndefined(props.itemId)
+            ? props.itemId
+            : !ListboxItem.isNullOrUndefined(props.id)
+                ? props.id
+                : props.value;
+    };
+
+    /**
+     * Gets the id to use for an listbox item by examining the associated Element
+     */
+    public static getItemIdFromElement = (element: Element): string => {
+        let idToUse: string = element.getAttribute("itemId");
+        if (!ListboxItem.isNullOrUndefined(idToUse)) {
+            return idToUse;
+        }
+
+        idToUse = element.getAttribute("id");
+        if (!ListboxItem.isNullOrUndefined(idToUse)) {
+            return idToUse;
+        }
+
+        return element.getAttribute("value");
+    };
+
+    public static isNullOrUndefined = (value: string): boolean => {
```

Comment:
Does this need to be a public static method? Can we just use `isNil` from lodash-es? https://lodash.com/docs#isNil

Reply:
Yes, got rid of that function entirely.